docs/developer-guides/native-token-reserve-management.md

```diff
@@ -0,0 +1,451 @@
+
+1. **On Cardano:** 
+
+- Creating your new token and setting up rules for how tokens are released over time. 
+- This involves setting up and managing a token management contract on the Cardano blockchain, enabling proper observability on your partner chain. 
+
+2. **On your partner chain:** 
+
+- Connecting your partner chain to track the token and enable token operations across both chains. 
+- This involves following a step-by-step process for interacting with both the Cardano blockchain and your partner chain.
+
+After you complete the steps in this guide, you will have a fully functioning token system that works across Cardano and your partner chain.
+
+![Native token management contract](native-token-mgt-contract.png "Native token management contract")
+
+## Prerequisites
+
+- **Cardano Node Setup:** A running Cardano node, including Ogmios and Kupo, to interact with the Cardano blockchain.
+
+- **Partner Chain Node Setup:** A running partner chain node.
+
+- **Development Environment:** Haskell and related tools installed for running scripts.
+
+- **Native Token Creation Knowledge:** Familiarity with creating Cardano native tokens.
+
+- **Command-Line Tools:** `pc-contracts-cli` ([partner-chains-smart-contracts release binary](https://github.com/input-output-hk/partner-chains-smart-contracts/releases)) 
+
+## Rewards schemes
+
+Please note that rewards schemes for token release is the responsibility of the Chain Builder. 
+
+# Part One: Initialization
+
+Initialization includes creating the native token on Cardano, writing the `VFunction`, running the commands `init-reserve-management` and `reserve-create` at the `pc-contracts-cli` side. 
+
+## 1. Setting up your token on Cardano
+
+### 1.1 Creating a native token
+
+Objective: Create a Cardano native token within your partner chain ecosystem.
+
+#### Steps
+
+This step is outside the scope of partner chains and requires following the standard procedure for creating Cardano native tokens. 
+Please see: [Minting Native Assets](https://developers.cardano.org/docs/native-tokens/minting/) on the Cardano Developer Portal for details. 
+Generally speaking, however, the process includes these steps:
+
+1. Defining the token policy: establishing the monetary policy script for your token.
+
+2. Minting the token: using the Cardano CLI to mint the token according to your policy.
+
+3. Recording the token details:
+
+   - Policy ID: The identifier of the token policy.
+
+   - Asset Name: The name of the token in hexadecimal.
+
+### 1.2 Configuring the token release schedule
+
+#### 1.2.1 Implementing V(t) release function
+
+**Understanding the `VFunction`:**
+
+- The `VFunction` defines the schedule and amount of tokens `A` that move from the reserve supply to the circulating supply over time.
+
+- `VFunction` is a minting policy that has to accept 2 parameters (or more if you like).
+
+   - The first parameter is the `VFunction` Redeemer (which can be anything)
```

Comment:
VFunction is called by our code. We decide what redeemer is, don't we?

How is user supposed to write VFunction when the first parameter "can be anything"?